src/Raven.Client/Documents/Queries/Timings/QueryTimings.cs

```diff
@@ -5,10 +5,19 @@
 
 namespace Raven.Client.Documents.Queries.Timings
 {
+    /// <summary>
+    ///     Representation of query timings.
+    /// </summary>
```

Comment:
Maybe let's elaborate more what exactly is included there. For example in the docs (https://ravendb.net/docs/article-page/6.0/Csharp/client-api/session/querying/debugging/query-timings) we say:

> detailed stats of the time spent by RavenDB on each part of the query

Also referencing  that docs article would be nice

Reply:
Changed, I think it's enough to say this contains total time spent on server-side query, and time spent on each particular query part